pkgs/top-level/all-packages.nix

```diff
@@ -40802,6 +40802,8 @@ with pkgs;
 
   terraform-backend-git = callPackage ../applications/networking/cluster/terraform-backend-git { };
 
+  terraform-cloud-agent = callPackage ../tools/infrastructure/terraform-cloud-agent {};
```

Comment:
heads up that new packages should be added to `pkgs/by-name` (new structure) and this line removed

Reply:
Addressed in [586c570](https://github.com/NixOS/nixpkgs/pull/294366/commits/586c5704fa7586291bcb43c4bf20975c91a426dd).